src/main/java/core/basesyntax/service/ReaderService.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+
+public interface ReaderService {
+    List<FruitTransaction> read(String fileName);

```

Comment:
let's create separate interface that will be parsing file lines to transactions, right know this method violates Single Responsibility principle